tests/org.jboss.reddeer.junit.test/src/org/jboss/reddeer/junit/test/internal/requirement/TestRequirementA.java

```diff
@@ -48,6 +48,11 @@ public Annotation getDeclaration() {
 	@Override
 	public void cleanUp() {
 		// TODO Auto-generated method stub
-		
+	}
+	
+	@Override
+	public long getPriority() {
+		// TODO Auto-generated method stub
+		return Requirement.super.getPriority();
```

Comment:
Well, either dont use this, because the requirement interface has default implementation and that one is used, so this is useless. Or specify own value.